frame/treasury/src/lib.rs

```diff
@@ -393,6 +396,37 @@ pub mod pallet {
 				.map_err(|_| Error::<T, I>::TooManyApprovals)?;
 			Ok(())
 		}
+
+		/// Force a previously approved proposal to be removed from the approval queue.
+		/// The original deposit will no longer be returned.
+		///
+		/// May only be called from `T::RejectOrigin`.
+		///
+		/// # <weight>
+		/// - Complexity: O(A) where `A` is the number of approvals
+		/// - Db reads and writes: `Approvals`
+		/// # </weight>
+		///
+		/// Errors:
+		/// - `ProposalNotApproved`: Proposal has not been approved.
```

Comment:
Sorry, but this isn't quite informative. You need to think about what this means when someone reads this. Which proposal wasn't approved? What are the causes for this error? These are questions that someone who encountered the `ProposalNotApproved` error would really want to know about, and attempt to fix it themselves.

There are two reasons why this can happen -- either the proposal exists but was not approved, or that a proposal with the specified `proposal_id` doesn't exist altogether. These two bits of information are very helpful for users who encounter this error and are trying to look for the next steps in resolving it.

Reply:
Does this work:

```
The `proposal_id` supplied was not found in the approval queue, i.e., the proposal has not been approved. 
This could also mean the proposal with the supplied `proposal_id` does not exist altogether, 
thus there is no way it would have been approved in the first place.
```

Reply:
Hi @KiChjang, I've modified the description.